README.md

```diff
@@ -68,6 +68,11 @@ defmodule MyApp.Endpoint do
 end
 ```
 
+Wildcards can be used in the origins specification, by using regex:
```

Comment:
I intentionally started the line with "Wildcards", as a kind of header of the paragraph. Wildcards is what you want to achieve, regex is the means to this end, that was my line of thought. Should I give the paragraph it's own header, and use your phrases?